[2021-10-09 06:39:08,294] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: v2_context.t3 2021-10-09T06:39:04.558657+00:00 [queued]>
[2021-10-09 06:39:08,301] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: v2_context.t3 2021-10-09T06:39:04.558657+00:00 [queued]>
[2021-10-09 06:39:08,301] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-10-09 06:39:08,301] {taskinstance.py:1095} INFO - Starting attempt 1 of 1
[2021-10-09 06:39:08,301] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-10-09 06:39:08,311] {taskinstance.py:1114} INFO - Executing <Task(PythonOperator): t3> on 2021-10-09T06:39:04.558657+00:00
[2021-10-09 06:39:08,313] {standard_task_runner.py:52} INFO - Started process 4827 to run task
[2021-10-09 06:39:08,316] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'v2_context', 't3', '2021-10-09T06:39:04.558657+00:00', '--job-id', '361', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/v2_context.py', '--cfg-path', '/tmp/tmp3ewtewmm', '--error-file', '/tmp/tmpjsabpsz1']
[2021-10-09 06:39:08,317] {standard_task_runner.py:77} INFO - Job 361: Subtask t3
[2021-10-09 06:39:08,354] {logging_mixin.py:109} INFO - Running <TaskInstance: v2_context.t3 2021-10-09T06:39:04.558657+00:00 [running]> on host a6cf1bcf3ec3
[2021-10-09 06:39:08,403] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=v2_context
AIRFLOW_CTX_TASK_ID=t3
AIRFLOW_CTX_EXECUTION_DATE=2021-10-09T06:39:04.558657+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-09T06:39:04.558657+00:00
[2021-10-09 06:39:08,404] {logging_mixin.py:109} INFO - {'conf': <airflow.configuration.AirflowConfigParser object at 0x7f9b56ea7eb0>, 'dag': <DAG: v2_context>, 'dag_run': <DagRun v2_context @ 2021-10-09 06:39:04.558657+00:00: manual__2021-10-09T06:39:04.558657+00:00, externally triggered: True>, 'ds': '2021-10-09', 'ds_nodash': '20211009', 'execution_date': DateTime(2021, 10, 9, 6, 39, 4, 558657, tzinfo=Timezone('+00:00')), 'inlets': [], 'macros': <module 'airflow.macros' from '/usr/local/lib/python3.8/dist-packages/airflow/macros/__init__.py'>, 'next_ds': '2021-10-09', 'next_ds_nodash': '20211009', 'next_execution_date': DateTime(2021, 10, 9, 6, 39, 4, 558657, tzinfo=Timezone('+00:00')), 'outlets': [], 'params': {}, 'prev_ds': '2021-10-09', 'prev_ds_nodash': '20211009', 'prev_execution_date': DateTime(2021, 10, 9, 6, 39, 4, 558657, tzinfo=Timezone('+00:00')), 'prev_execution_date_success': <Proxy at 0x7f9b51987600 with factory <function TaskInstance.get_template_context.<locals>.<lambda> at 0x7f9b519f3ca0>>, 'prev_start_date_success': <Proxy at 0x7f9b519879c0 with factory <function TaskInstance.get_template_context.<locals>.<lambda> at 0x7f9b5198cdc0>>, 'run_id': 'manual__2021-10-09T06:39:04.558657+00:00', 'task': <Task(PythonOperator): t3>, 'task_instance': <TaskInstance: v2_context.t3 2021-10-09T06:39:04.558657+00:00 [running]>, 'task_instance_key_str': 'v2_context__t3__20211009', 'test_mode': False, 'ti': <TaskInstance: v2_context.t3 2021-10-09T06:39:04.558657+00:00 [running]>, 'tomorrow_ds': '2021-10-10', 'tomorrow_ds_nodash': '20211010', 'ts': '2021-10-09T06:39:04.558657+00:00', 'ts_nodash': '20211009T063904', 'ts_nodash_with_tz': '20211009T063904.558657+0000', 'var': {'json': None, 'value': None}, 'yesterday_ds': '2021-10-08', 'yesterday_ds_nodash': '20211008', 'templates_dict': None}
[2021-10-09 06:39:08,404] {python.py:151} INFO - Done. Returned value was: None
[2021-10-09 06:39:08,414] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=v2_context, task_id=t3, execution_date=20211009T063904, start_date=20211009T063908, end_date=20211009T063908
[2021-10-09 06:39:08,447] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-09 06:39:08,473] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
